src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -37,11 +40,16 @@ GlobalAppSettings::GlobalAppSettings() :
     _alwaysShowTabs{ true },
     _initialRows{ DEFAULT_ROWS },
     _initialCols{ DEFAULT_COLS },
+    _initialX{ 0 },
+    _initialY{ 0 },
+    _useDefaultInitialX{ true },

```

Comment:
Shouldn't these be false by default unless set by the load of preferences?

Reply:
I have changed the flag to isInitialX/YSet, and the default value is false. For the _initialX/Y, we actually won't use this default value, I wonder if you think we can just initialize them with {} ?

Reply:
I just meant `false` instead of `true` which it looks like you did in the latest revision.